### PR TITLE
Add io.github.orontee.Argos

### PR DESCRIPTION
**Pull request by orontee**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: io.github.orontee.Argos.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+    "app-id" : "io.github.orontee.Argos",
+    "runtime" : "org.gnome.Sdk",
+    "runtime-version" : "41",
```
Comment (hfiguiere) [inline io.github.orontee.Argos.json:4]:
```suggestion
    "runtime-version" : "42",
```

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: io.github.orontee.Argos.json (hunk 1 above)
Comment (orontee) [reply io.github.orontee.Argos.json:4]: Looks good to me. Thanks! Applied upstream, https://github.com/orontee/argos/commit/ab31a584b6ab8a68f7772d84536b78c89f6abced

**Comment by hfiguiere** [comment]
bot, build io.github.orontee.Argos

**Comment by hfiguiere** [comment]
this icon in the .desktop file is incorrect. It should be the value of `$FLATPAK_ID`.

Build complains about `io.github.orontee.Argos.desktop:   using stock icon media-optical`

**Comment by orontee** [comment]
bot, build io.github.orontee.Argos

**Comment by orontee** [comment]
I installed and tested from the testing repository; LGTM.

**Comment by barthalion** [comment]
I'd suggest adding some kind of prompt to notify user that Mopidy server is unreachable. Not a blocker, though.

/merge

**Comment by orontee** [comment]
> notify user that Mopidy server is unreachable.

@barthalion Thanks for the feedback, tracked with https://github.com/orontee/argos/issues/43.

**Pull request closed without merging**